src/components/menu/js/menuController.js

```diff
@@ -118,14 +118,17 @@ function MenuController($mdMenu, $attrs, $element, $scope, $mdUtil, $timeout, $r
     triggerElement = triggerElement || (ev ? ev.target : $element[0]);
     triggerElement.setAttribute('aria-expanded', 'true');
     $scope.$emit('$mdMenuOpen', $element);
+    var hasBackdrop = getAttr('useBackdrop', true);
```

Comment:
Same functionality, without the new method 
```js
angular.isDefined($attrs.useBackdrop) ? $attrs.useBackdrop : true;
```

But in case of using it as an attribute, it would better call it `md-disable-backdrop`. Because `use-backdrop="true"` is actually not making a sense.